Formula/dnsviz.rb

```diff
@@ -15,28 +15,30 @@ class Dnsviz < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "bind" => :test
```

Comment:
How is this used in the test ? is dnsviz calling it somehow ?

Reply:
yes.

https://github.com/dnsviz/dnsviz/blob/master/dnsviz/commands/probe.py#L607

It is an optional dependency (user is notified that they should install it if they want to use the feature that requires it), but the test requires it.